src/azure-cli/azure/cli/command_modules/marketplaceordering/generated/commands.py

```diff
@@ -0,0 +1,31 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-statements
+# pylint: disable=too-many-locals
+# pylint: disable=bad-continuation
+# pylint: disable=line-too-long
+
+from azure.cli.core.commands import CliCommandType
+from ..generated._client_factory import cf_marketplace_agreement
+
+
+marketplaceordering_marketplace_agreement = CliCommandType(
+    operations_tmpl='azure.mgmt.marketplaceordering.operations._marketplace_agreements_operations#MarketplaceAgreementsOperations.{}',
+    client_factory=cf_marketplace_agreement,
+)
+
+
+def load_command_table(self, _):
+
+    with self.command_group(
+        'term', marketplaceordering_marketplace_agreement, client_factory=cf_marketplace_agreement, is_experimental=True
+    ) as g:
+        g.custom_show_command('show', 'term_show')
+        g.custom_command('accept', 'term_accept')
```

Comment:
How to cancel a marketplace term?

Reply:
cancel via portal, so far service doesn't expose api to client to cancel except portal